scripts/generateSvg.ts

```diff
@@ -0,0 +1,89 @@
+import fs from 'fs';
```

Comment:
sorry max - this should probably be `generate-logger-preview`? or be more generic? currently it implies it's generic but it's not